LICENSE

```diff
@@ -1,116 +1,8 @@
-CC0 1.0 Universal
```

Comment:
I copied the `LICENSE` from the JVM project, because I saw a difference between `LICENSE` and `COPYING` and I also brought it `COPYING`. Not sure why, thought it makes sense to have it identical to the JVM project, but I'm not a lawyer.

I can revert this change if you want, no problem.

If you want, I can revert the file.